src/router/bundle/bundleHandler.ts

```diff
@@ -146,6 +147,32 @@ export default class BundleHandler implements BundleHandlerInterface {
             }
         }
 
+        const authAndFilterReadPromises = requests.map((request, index) => {
+            if (['read', 'vread', 'history-type', 'history-instance', 'search-type'].includes(request.operation)) {

```

Comment:
Can you do `system-search` in bundles?

Reply:
Technically you can, but in our implementation we only support transactions that contains CRUD requests. So in this particular use case we would only ever be filtering `read` responses. However, I'll add `search-system`  and `history-system` to the list to be complete.

https://github.com/awslabs/fhir-works-on-aws-routing/blob/496350f499776d5ebec6a6d1a601e7def26178a3/src/router/bundle/bundleParser.ts#L72